application/base.py

```diff
@@ -49,121 +47,3 @@ def handle_errors(schema, errors, obj):  # noqa
                      .format(errors))
     current_app.logger.error(error_message)
     raise ValueError(error_message)
```

Comment:
Is that still used? If yes we should move it into a different file, base.py is a horrible name :D

Reply:
Ah okay that's where the base schema still lives. I think base_schema.py would be better but we can ignore that for now

Reply:
done